src/ios/PurchasesPlugin.swift

```diff
@@ -43,7 +43,8 @@ import RevenueCat
 
         self.purchases = Purchases.configure(apiKey: apiKey,
                                              appUserID: appUserID,
-                                             observerMode: observerMode,
+                                             purchasesAreCompletedBy: observerMode ?
+                                             PurchasesAreCompletedBy.myApp : PurchasesAreCompletedBy.revenueCat,
```

Comment:
Nitpick but I wonder if we can do this. Might look nicer. 
```suggestion
                                            .myApp : .revenueCat,
```